cli/index.js

```diff
@@ -54,17 +53,16 @@ const outputMode = cli.flags.output || 'pretty';
 const outputPath = cli.flags.outputPath || 'stdout';
 const flags = cli.flags;
 
-const driver = new ChromeProtocol();
-
 // set logging preferences
+flags.logLevel = 'info';
```

Comment:
log level is also a bit weird because it's global state, we're passing it into `lighthouse` to set it, and depending on it out here for logging printer status. Not sure if there's a clean fix for that, though, short of passing a logger into lighthouse